modules/servers/openprovider/configs.example.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
it's better to use .yml format

Reply:
To use yml format we need to install additional libs for process it. But I want to make it not so complicated